### DISCUSSION TITLE
Photo Storage Query

### DISCUSSION THREAD

**Discussion by rxerium** [immich-app/immich#1975] [Q&A] [answered]
Hi there,

How does the photo storage work with Immich? I'd like Immich to point to a network share which contains all the photos/videos, is this possible?

Thanks

**Comment by alextran1502** [comment]
Hey, this is not possible. You will have to upload your photo to Immich so that Immich and process all the information and generate additional files needed for other functions of the application

**Reply by bo0tzz** [reply]
You can of course use a network share as storage for Immich's library folder.

**Reply by rxerium** [reply]
I'm confused, one say its possible and the other say its not?

**Reply by bozzfozz** [reply]
import is only possible via cli but as a location where immich saves the photos you can select network share

**Reply by bozzfozz** [reply]
if you need something like that
feat(server): import location #1319

**Reply by bo0tzz** [reply, marked as answer]
> I'm confused, one say its possible and the other say its not?

You can store Immich's library on a network share, but you can't use a a folder that already has things in it.

**Reply by rxerium** [reply]
Thanks, I'll have a play around and will let you know if I have any issues.
